features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -222,724 +231,164 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
     };
 
     /**
-     * Resolves the tab panes based on the application config.
-     *
-     * @returns Resolved tab panes.
+     * Todo Remove this mapping and fix the backend.
      */
-    const resolveTabPanes = (): ResourceTabPaneInterface[] => {
-        const panes: ResourceTabPaneInterface[] = [];
-        const extensionPanes: ResourceTabPaneInterface[] = [];
+    const mapProtocolTypeToName = ((type: string): string => {
+        let protocolName: string = type;
 
-        if (!tabPaneExtensions && applicationConfig.editApplication.extendTabs
-            && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_OIDC
-            && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_PASSIVE_STS
-            && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_SAML) {
-            return [];
+        if (protocolName === "oauth2") {
+            protocolName = SupportedAuthProtocolTypes.OIDC;
+        } else if (protocolName === "passivests") {
+            protocolName = SupportedAuthProtocolTypes.WS_FEDERATION;
+        } else if (protocolName === "wstrust") {
+            protocolName = SupportedAuthProtocolTypes.WS_TRUST;
+        } else if (protocolName === "samlsso") {
+            protocolName = SupportedAuthProtocolTypes.SAML;
```

Comment:
Shall we use constants for the string literals here? better if we can define the constants in a reusable manner.

Reply:
This is an existing code block. https://github.com/wso2/identity-apps/blob/05605b9367987c95ce06add63ce0e95494a90b2d/features/admin.applications.v1/components/edit-application.tsx#L790-L798